sdk/storage/azure-storage-blob/azure/storage/blob/_upload_helpers.py

```diff
@@ -136,6 +142,10 @@ def upload_block_blob(  # pylint: disable=too-many-locals, too-many-statements
                 cek, iv, encryption_data = generate_blob_encryption_data(
```

Comment:
azure\storage\blob\_upload_helpers.py:142: error: Incompatible types in assignment (expression has type "Optional[str]", variable has type "str")  [assignment]